### ISSUE TITLE
Subtitles stops working randomly

### ISSUE THREAD

**Issue by Dis90** [asciidisco/plugin.video.netflix#62]
Like some videos I can see subtitles for 3 minutes and then nothing but in some videos subtitles stops working after 1 minute and then shows randomly.

I've been watching this show '13 reasons why' and it seems like plugin is showing subtitles only for lines what narrator speaks.

Long debug log with ffmpeg and video components logging: http://paste.ubuntu.com/24422205/

Ubuntu: 14.04
Kodi: 2f67109
Inputstream: a7e84de
Netflix-plugin: v0.11.11

**Comment by Uukrull** [comment]
That's because the inputstream.adaptive addon only takes into account lines with the "span" tag and ignores the rest. This commit from me fixes that and adds support for italics:

https://github.com/Uukrull/inputstream.adaptive/commit/3fe7a6656d3f2e3debb19204311e72bce6cae95c

**Comment by Dis90** [comment]
Thanks for quick reply. I compiled your agile_fix branch of inputstream.adaptive and subtitles seems to work now.

**Comment by asciidisco** [comment]
@Uukrull @liberty-developer Do you know if this resolved in the Kodi Nightlies (and included inputstream) now?

**Comment by Uukrull** [comment]
Yes, it's solved. The only problem now is that, sometimes, extras lines are added:

![](http://i.imgur.com/lzWxAuA.png)

Removing this [line](https://github.com/peak3d/inputstream.adaptive/blob/master/src/parser/TTML.cpp#L236) fixes the issue:

![](http://i.imgur.com/0h44vzC.png)

**Comment by asciidisco** [comment]
@Uukrull Thank you. I´ll send your comment to @liberty-developer

**Comment by asciidisco** [comment]
Issue is reported, but may take some time to find its way into inputstream. Renaming this one for the time being.
